benchmarks/spack/repo/packages/arepo/package.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from spack.package import *
+
+class Arepo(MakefilePackage):
+    """Arepo is a massively parallel gravity and hydrodynamics code for astrophysical simulations."""
+    homepage = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    git = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    
+    
+    version("master", branch="master")
+   
+
+    depends_on('mpi')
+    depends_on('gsl')
+    depends_on('hdf5')
```

Comment:
Should hdf5 be an optional dep added only when hdf5 support is requested?

Reply:
Exactly, I have already added it as a variant.

Reply:
I think you also need to add something like
```
depends_on('hdf5', when='+hdf5')
```